jx_sqlite/setop_table.py

```diff
@@ -119,13 +119,13 @@ def place(parent_doc_details):
             sql_selects.append(sql_select + " AS " + _make_column_name(column_number))
             if nested_path != ".":
                 sql_select = alias + "." + quote_table(ORDER)
-                sql_selects.append(sql_select + " AS " + _make_column_name(column_number))
+                sql_selects.append(sql_select + " AS " + _make_column_name(column_number+1))
```

Comment:
it won't matter to logic of query, but for symmetry i think it is important as `column_number=len(sql_selects)` and if order is used as column in select of the sql query, we don't want it to have same alias as of  previous column's (here uid). Obvious, it won't change meaning of the query.